### DISCUSSION TITLE
Release new version of selectors crate to crates.io

### DISCUSSION THREAD

**Discussion by emirror-de** [servo/servo#28666] [Q&A]
Dear servo contributers and maintainers,

is it possible to release a new version of the selectors crate to crates.io containing the updated cssparser dependency?
I am using it for one of my projects and need to wait until the updated version is released before I can continue.

Best regards,
Lewin

**Comment by jdm** [comment]
I'm pretty sure that the official home of the selectors crate is now https://searchfox.org/mozilla-central/source/servo/components/selectors, and we don't have an automated mechanism to synchronize changes from there any more, sadly.

**Reply by emirror-de** [reply]
Hm ok, so what does that mean? Is there a possibility to change that, or will there be no new release anymore?

**Reply by jdm** [reply]
@emilio How do new versions of the selectors crate get released these days?
